modules/update-upgrading-sro.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-clusters-sro.adoc
+
+:_content-type: PROCEDURE
+[id="updating-clusters-sro_{context}"]
+= Running a preflight check for the SRO
+
+The following is an example procedure for checking the compatibility of a kernel module version before upgrading a cluster.
+
+.Prerequisites
+
+* You have a running {product-title} cluster.
+* You installed the OpenShift CLI (oc).
+* You are logged into the OpenShift CLI as a user with cluster-admin privileges.
+* You installed the SRO.
+
+.Procedure
+
+. Create the following CRD and save the YAML as `prevalidation.yaml`.
++
+[source,yaml]
+----
+apiVersion: sro.openshift.io/v1beta1
+kind: PreflightValidation
+metadata:
+ name: preflight
+ namespace: preflight
+spec:
+ updateImage: quay.io/openshift-release-dev/ocp-release@sha256:f7f252c39b64601c8ac3de737a584ba4f6016b1f4b17801d726ca2fd15492878 <1>
+----
+<1> Specify the name of the upgrade image here
```

Comment:
It is not precisely a name, maybe:
"Specify the upgrade image here"?

Reply:
ok. Will go for 
Specify the update image here
as the propertry nems is "updateImage":